hca/dss/util/__init__.py

```diff
@@ -58,7 +58,13 @@ def hardlink(source, link_name):
         if source_stat.st_dev != dest_stat.st_dev or source_stat.st_ino != dest_stat.st_ino:
             raise
     except OSError as e:
-        if e.errno == errno.EMLINK:
+        # See https://docs.python.org/3/library/errno.html
+        copy_on_error = [
```

Comment:
We typically use tuples for immutable sequences that are used with `in`. Not for performance reasons but to more clearly indicate intent.

Reply:
Changed to tuple